lya/analysis.py

```diff
@@ -18,9 +30,24 @@ def residuals(pars, data):
     pars : array-like
         [alpha, beta, gamma] parameters.
 
-    data : Data object
+    data : Data object (string)
         Data to be compared with model.
     """
+   
+    file=np.loadtxt(data)
+    k=file[:,0]
+    pk=file[:,1]
+    pkerr=file[:,2]
```

Comment:
You can remove these lines. The current implementation of the data object has explicit attributes which you can call like this:

`data.k`, `data.P_k` and `data.P_k_unc`

Reply:
I think I'd like to stick with the `Data` attributes being `x` and `y` rather than `k` and `P_k`, just to remind us that there's nothing particular to cosmology about the exercise we're doing (will comment on your PR soon to this same effect @syasini ).  So @annahaynie you can use `data.x`, etc.